include/rocksdb/utilities/stackable_db.h

```diff
@@ -92,10 +92,6 @@ class StackableDB : public DB {
                    const WideColumns& columns) override {
     return db_->PutEntity(options, column_family, key, columns);
   }
-  Status PutEntity(const WriteOptions& options, const Slice& key,
-                   const AttributeGroups& attribute_groups) override {
-    return db_->PutEntity(options, key, attribute_groups);
-  }
```

Comment:
Oh sorry - I think we would still want keep this; I just commented on this chunk because this override was what made me think about read-only and secondary db's ;)